websites/K/Kemono.Party/presence.ts

```diff
@@ -0,0 +1,74 @@
+const presence = new Presence({
+		clientId: "1067711765770682388",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	let presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/mwpEkf3.png",
```

Comment:
Same as above & must be 512x512
```suggestion
		largeImageKey: "https://i.imgur.com/jdXZ8kV.png",
```